### DISCUSSION TITLE
App 置于前台时，收到消息缺少感知

### DISCUSSION THREAD

**Discussion by axelburks** [chanify/chanify#48] [Ideas]
打开 App，发送通知，收到通知后，无弹窗，无通知，缺少感知

**Comment by wizjin** [comment]
后面的版本会在打开 App时增加小红点的提示。

**Comment by wizjin** [comment]
@axelburks 小红点提示已添加到 v1.0.10 https://github.com/chanify/chanify-ios/issues/7
